cms/toolbar/utils.py

```diff
@@ -116,3 +118,24 @@ def get_toolbar_from_request(request):
     from .toolbar import EmptyToolbar
 
     return getattr(request, 'toolbar', EmptyToolbar(request))
+
+
+def get_object_edit_url(obj):
+    content_type = ContentType.objects.get_for_model(obj)
+
+    with force_language(get_current_language()):
```

Comment:
don't hardcode current language, pass language as a parameter and if none fallback to django's get_current_language instead of the cms one

Reply:
This applies to all three functions